docs/help/FAQ/10_Password_Complexity.rst

```diff
@@ -17,17 +17,32 @@ The default complexity is enforced in both :term:`PAM` and :term:`LDAP`.
 Complexity Rules
 ----------------
 
-  * 14 Characters or greater
+  * 15 Characters or greater
   * 1 Upper case letter
   * 1 Lower case letter
   * 1 Number
   * 1 Special character
   * No more than 2 repetitions of the same character
+  
+    * OK: ``aab``
+    * BAD: ``aaa``
+
+  * No more than 3 repetitions of a character from the same character class
+  
+    * OK: ``abc``
+    * BAD: ``abcd``
+
   * No more than 4 characters in a monotonic character sequence
+  
+    * OK: ``1b2c3d``
+    * BAD: ``aBcDe``
+
+  * Cannot contain your username
+  * Cannot contain items from your GECOS field (usually your full name)
+  * Must have more than 4 character changes from the old password
```

Comment:
I just tested this against the pam module's defaults (from a pry session in the acceptance tests).  The policy enforces the documentation for normal users changing their own passwords, but the root user can change a normal user's password by only a single character using either el6/`cracklib` and el7/`pw_quailty`.

Reply:
Well, sure. `root` can also edit `/etc/shadow` directly so I don't think we need to tell people that `root` has magic powers.